src/orders/utils/data.ts

```diff
@@ -25,6 +27,20 @@ export type OrderWithTotalAndTotalCaptured = Pick<
   "total" | "totalCaptured"
 >;
 
+export function getOrderWarehouses(order: OrderFulfillData_order) {
```

Comment:
Mark required parameters for this function to work properly and extend over it. At the moment it's too bounded to specific query data. `T extends { ... }` would be more appropriate